src/main/resources/assets/trafficcontrol/models/block/overhead_crossbuck.old

```diff
@@ -0,0 +1,170 @@
+{
```

Comment:
Please get rid of these .old files - that's what source control is for

Reply:
Removed. Forgot about those